library/CMakeLists.txt

```diff
@@ -191,6 +191,14 @@ if (APPLE OR UNIX)
   endif ()
 endif ()
 
+if(UNIX AND NOT APPLE)
+  option(F3D_LINUX_LIBRARY_LINK_ATOMIC "Link with libatomic" OFF)
```

Comment:
Add a mention of it in https://f3d.app/doc/user/LIMITATIONS_AND_TROUBLESHOOTING.html ?

https://github.com/f3d-app/f3d/blob/master/doc/user/LIMITATIONS_AND_TROUBLESHOOTING.md

Reply:
Done!

Reply:
perfect!